postgresql/database.go

```diff
@@ -84,79 +78,242 @@ func debugLog(query string, args []interface{}, err error, start int64, end int6
 	}
 }
 
-func init() {
+// Returns the underlying *sqlx.DB instance.
+func (s *source) Driver() interface{} {
```

Comment:
I think it's important not to break user's code intentionally, even for those who don't use a management tool besides `go get`, so for the current public version I still think we should stick to `Driver()` returning `*sql.DB` as it is now.

But here's a thing, we haven't decided if we're working on replacing the current version! so regarding to the `feature/sql` branch, I'd keep your proposal (`Driver()` returning `*sqlx.DB`) as I also find it very useful, working that way could lead to many different interesting solutions and that's one of the advantages of switching to `*sqlx.DB`.

If we don't find a way to avoid breaking user's code we'll probably need to release a `upper.io/v2/db`, and that would be great as there are some other breaking changes that I'd love to introduce. Or should we push them to `upper.io/db` (current feature/sql) and provide a `upper.io/v1/db` (old version)? Let's talk about that later, the important thing is to keep going forward while providing an easy solution for our current users in case we end up breaking things.

So yes, let's keep working in this branch with `*sqlx.DB`.